bedrock/mozorg/helpers/misc.py

```diff
@@ -37,6 +39,25 @@ def convert_to_high_res(url):
 
 
 @jingo.register.function
+def switch(name):
+    """A template helper that replaces waffle
+
+    * All calls default to True when DEV setting is True.
+    * If the env var is explicitly false it will be false even when DEV = True.
+    * Otherwise the call is False by default and True is a specific env var exists and is truthy.
+
+    For example:
+
+        {% if switch('dude-and-walter') %}
+
+    would check for an environment variable called `SWITCH_DUDE_AND_WALTER`. The string from the
+    `switch()` call is converted to uppercase and dashes replaced with underscores.
+    """
+    env_name = 'SWITCH_' + name.upper().replace('-', '_')
+    return config(env_name, default=settings.DEV, cast=bool)
```

Comment:
I did look at the implementation in decouple and it loads all it's stuff just once at startup, so since we already use it in settings and this instances is a singleton it should be equally efficient to use this as `os.environ` directly, plus we get the benefit of the boolean casting from strings, the easy default, and managing locally via a `.env` file if devs want/need to.